packages/openactive-integration-tests/test-data-generator/test-data-generator.js

```diff
@@ -93,6 +94,7 @@ const categoryOrFeature = /** @type {string} */(categoryOrFeatureUntyped); // ya
   // ## Create Test Data
   //
   // One for each seller x opportunity criteria
+  const harvestStartTimeOverride = DateTime.now().toISO();
```

Comment:
Problem #1. The generated test data had date constraints like `maxDate: null` because `harvestStartTime` was not provided (due to the global `HARVEST_START_TIME` not existing when running test-data-generator)